sdk/cognitiveservices/Vision.ComputerVision/src/Microsoft.Azure.CognitiveServices.Vision.ComputerVision.csproj

```diff
@@ -2,8 +2,7 @@
   <PropertyGroup>
     <AssemblyTitle>Microsoft Cognitive Services ComputerVision SDK</AssemblyTitle>
     <Description>This client library provides access to the Microsoft Cognitive Services ComputerVision APIs.</Description>
-    <Version>6.1.0-beta.1</Version>
-    <ApiCompatVersion>6.0.0</ApiCompatVersion>
```

Comment:
The `<ApiCompatVersion>` likely shouldn't be removed; this helps ensure that we don't introduce breaking changes when compared to the most recent the GA version.   Was this intentional because we're expecting breaking changes?

Reply:
It was intentional as it contains some breaking changes.